frostsnapp/lib/key_settings.dart

```diff
@@ -0,0 +1,82 @@
+import 'dart:async';
+import 'dart:collection';
+import 'dart:typed_data';
+import 'dart:io';
+import 'package:flutter/material.dart';
+import 'package:frostsnapp/animated_check.dart';
+import 'package:frostsnapp/device_action.dart';
+import 'package:frostsnapp/device_id_ext.dart';
+import 'package:frostsnapp/device_list_widget.dart';
+import 'package:frostsnapp/device_setup.dart';
+import 'package:frostsnapp/global.dart';
+import 'ffi.dart' if (dart.library.html) 'ffi_web.dart';
+import 'hex.dart';
+import "dart:developer" as developer;
+
+class KeySettingsPage extends StatelessWidget {
+  final FrostKey frostKey;
+
+  const KeySettingsPage({super.key, required this.frostKey});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(title: const Text('Key Settings')),
+        body: Center(
+            child: Padding(
+          padding: EdgeInsets.all(8.0),
+          child: KeySettingsContent(frostKey: frostKey),
+        )));
+  }
+}
+
+class KeySettingsContent extends StatelessWidget {
+  final FrostKey frostKey;
```

Comment:
Having a key settings page is a fine idea but not really the point of this PR. It would make sense to see a list of devices that had key shares for example. But the `frostKey` parameter isn't being used here.

I think the simpler approach is just to have a `Show devices` button next to `New Key` on the key list which brings up the device list and has a `⚙` on each card where you can do the renaming workflow and show backups.

Reply:
Half fixed, still labeled as deviceSettings in code, need to add `⚙` and popup.